docs/examples/app-the-new-thing/src/interbit/control/index.js

```diff
@@ -0,0 +1,130 @@
+// © 2018 BTL GROUP LTD -  This package is licensed under the MIT license https://opensource.org/licenses/MIT
+const {
+  rootCovenant,
+  coreCovenant: {
+    actionCreators: { addToAcl },
+    redispatch
+  }
+} = require('interbit-covenant-tools')
+const Immutable = require('seamless-immutable')
+
+const { CHAIN_ALIASES, COVENANTS } = require('./constants')
+
+const actionTypes = {
+  MEOW: 'MEOW'
+}
+
+const actionCreators = {
+  meow: whoMeowed => ({
+    type: actionTypes.MEOW,
+    payload: {
+      whoMeowed
+    }
+  })
+}
+
+const initialState = Immutable.from({
+  // Not required but handy for troubleshooting
+  chainMetadata: { chainName: `Template application - control chain` },
+
+  // Where to find required Interbit services
+  // For security, this information is maintained in the control chain and
+  // shared via a read join to the public chain
+  interbitServices: {
+    // Using shared as a marker to indicate data shared with other chains
+    shared: {
+      // Required for cAuth
+      accounts: {
+        serviceEndpoint:
+          process.env.INTERBIT_ACCOUNTS_URL || 'http://localhost:3025'
+      },
+      // Required for deployment
+      projects: {
+        serviceEndpoint:
+          process.env.INTERBIT_PROJECTS_URL || 'http://localhost:3035'
+      }
+    }
+  },
+
+  // Information required for creating private chains using the
+  // Public/Private/Control (PPC) model
+  privateChainHosting: {},
+  cats: {}
+})
```

Comment:
Add a comment for cats. This is new chain state added to the control chain, not part of the PPC model.

**** METADATA LEAKAGE ***** THIS NEEDS TO BE FIXED ASAP!

I want to stop this repurposing of the control chain NOW, before users do it. We should have a 'shared' chain in the template specifically for this purpose to avoid polluting the PPC model. Anyone who loads the control chain will see every private chain ID for every user, will know how many use accounts have been created, etc., etc. That is a real bad thing!